pkg/pxc/backup/volume.go

```diff
@@ -8,7 +8,17 @@ import (
 )
 
 // NewPVC returns the list of PersistentVolumeClaims for the backups
-func NewPVC(cr *api.PerconaXtraDBClusterBackup) *corev1.PersistentVolumeClaim {
+func NewPVC(cr *api.PerconaXtraDBClusterBackup, cluster *api.PerconaXtraDBCluster) *corev1.PersistentVolumeClaim {
+	// Copy from the original labels to the backup labels
```

Comment:
Instead of coping these, there is this function `k8s.io/apimachinery/pkg/labels.Merge()`  that will merge two sets of labels, maybe you can use it.